tst/LargeAlgebras.tst

```diff
@@ -0,0 +1,61 @@
+gap> START_TEST( "LargeAlgebras.tst");
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236000));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 60 ], [ 2, 29, 0 ], [ 2, 31, 0 ] ], [ [ 0, 0, 0 ], [ 60, 0 ], [ 0 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[ [ infinity, 2 ] ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236001));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 0 ], [ 2, 29, 60 ], [ 2, 31, 0 ] ], [ [ 0, 0, 0 ], [ 60, 0 ], [ 0 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[  ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236002));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 60 ], [ 2, 29, 0 ], [ 2, 31, 0 ] ], [ [ 60, 0, 60 ], [ 0, 60 ], [ 0 ] ] 
+ ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[  ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236003));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 0 ], [ 2, 29, 60 ], [ 2, 31, 0 ] ], [ [ 0, 60, 0 ], [ 0, 0 ], [ 0 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);               
+[ [ 5, 2 ] ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236007));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 0 ], [ 2, 29, 60 ], [ 2, 31, 0 ] ], [ [ 0, 0, 0 ], [ 60, 0 ], [ 60 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[ [ 5, 2 ] ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236008));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 0 ], [ 2, 29, 0 ], [ 2, 31, 0 ] ], [ [ 0, 0, 0 ], [ 0, 60 ], [ 0 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[ [ infinity, 2 ] ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236009));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];                    
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 0 ], [ 2, 29, 60 ], [ 2, 31, 0 ] ], [ [ 0, 60, 0 ], [ 0, 0 ], [ 60 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[  ]
+
+#
+gap> STOP_TEST( "LargeAlgebras.tst", 1 );
```

Comment:
You commit message now says
```
Modify test for large algebras with four generated Galois groups
Drop some of the calculations to reduce the time and fix two typos
```
but since you are amending previous commit, it can be just as before:
```
Add test for large algebras with four generated Galois groups
```
since with `--amend` you make the history look like the previous commit never existed.
So, when you will be amending it next time, please edit it.

I have a suggestion for the next amendment: let's modify each block as follows: 

```
# gap> QG:=GroupRing(Rationals,SmallGroup(1920,236008));;
# gap> W:=WedderburnDecompositionInfo(QG);;
# gap> A:=W[Length(W)];
gap A:= [ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 0 ], [ 2, 29, 0 ], [ 2, 31, 0 ] ], [ [ 0, 0, 0 ], [ 0, 60 ], [ 0 ] ] ];;
gap> LocalIndicesOfCyclotomicAlgebra(A);
[ [ infinity, 2 ] ]
```
This way, we can quickly arrive to a passable test, and get an idea of timing and coverage. We will keep around the comments which will explain how the input was produced. Then we may decide to uncomment some of them back to have a real `WedderburnDecompositionInfo` run.

Reply:
@angeldelriomateos so what about this suggestion? Fixing `A` will also help to avoid diffs in  GAP 4.9. This seems a good test with potential to increase coverage and prevent more bugs from going undetected, so it's tempting to merge it now in order to test subsequent changes more thoroughly.

Reply:
> @angeldelriomateos so what about this suggestion? Fixing `A` will also help to avoid diffs in GAP 4.9. This seems a good test with potential to increase coverage and prevent more bugs from going undetected, so it's tempting to merge it now in order to test subsequent changes more thoroughly.

Ok. Go ahead

Reply:
@angeldelriomateos OK, I've pushed a fixup to your branch. It still seems long though...

Reply:
@angeldelriomateos leave it to me for now then - I will be doing a lot of force-pushes, so may be easier for you to not to touch this branch for now.